Source/WebCore/dom/TextEncoder.h

```diff
@@ -45,7 +45,6 @@ class TextEncoder : public RefCounted<TextEncoder> {
     RefPtr<Uint8Array> encode(String&&) const;
     EncodeIntoResult encodeInto(String&&, Ref<Uint8Array>&& destination);
 private:
-    TextEncoder() { };
```

Comment:
Deleting this isn’t right, because then the `TextEncoding` default constructor will be public instead of private. So please add this back with `= default`.

Reply:
ops, good catch, thanks!